src/public/style.css

```diff
@@ -1,12 +1,72 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather|Open+Sans');
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Open Sans', sans-serif;
+}
+
 body {
-  background-color: #d9e6fc;
+  font-family: 'Merriweather', serif;
+}
+
+
+
+header {
+  margin: 0 !important;
+  display: block;
+}
+
+div.page {
+    background-color: lightgrey;
+    }
+
+div.title {
+  position: static;
+}
+
+div.content-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-content: flex-start;
+  }
+
+main {
+  padding: 20px 20px 20px 20px;
```

Comment:
If padding on all sides is the same, typically we just use the shorthand `padding: 20px;`. That will apply equal padding all around. Likewise, `padding: 20px 10px;` would apply 20px padding to top and bottom, 10px to left and right.